_data/simple-icons.json

```
@@ -4621,6 +4621,11 @@
             "hex": "782A90",
             "source": "https://github.com/vert-x3/.github/blob/1ad6612d87f35665e50a00fc32eb9c542556385d/workflow-templates/vertx-favicon.svg"
         },
+        {
+            "title": "Ecosia",
+            "hex": "5DD25E",
+            "source": "https://www.ecosia.org"
```

```suggestion
            "source": "https://www.ecosia.org",
            "guidelines": "https://www.ecosia.org/imprint"
```
Their website seems to have been updated